docs_app/content/deprecations/resultSelector.md

```diff
@@ -0,0 +1,70 @@
+# ResultSelector Parameter
+
+Some operator supported a resultSelector argument that acted as mapping function on the result of that operator.
+The same behavior can be reproduced with the `map` operator, therefore this argument became deprecated.
+
```

Comment:
FYI: The reason this deprecation was added is because while having a resultSelector was good for performance during secondary mapping, in cases where secondary mapping wasn't necessary, it caused us to have to retain the outer value in memory longer than necessary and increased memory pressure.

It's also possible that we might reverse the decision on some of these where that isn't necessarily true.. `fromEvent` or `combineLatest` for example.